Application/Education/Certificate/Generator/Service/DataCertificate/IDataEZSH.php

```diff
@@ -0,0 +1,514 @@
+                    $Data->getTblSchoolTypeSecondary(), null, true);
+                if (!$Data->getCertificateLevelAllByCertificate($tblCertificate)) {
+                    if (($tblLevel = Division::useService()->getLevelBy($Data->getTblSchoolTypeSecondary(), '7'))) {
+                        $Data->createCertificateLevel($tblCertificate, $tblLevel);
+                    }
+                    if (($tblLevel = Division::useService()->getLevelBy($Data->getTblSchoolTypeSecondary(), '8'))) {
+                        $Data->createCertificateLevel($tblCertificate, $tblLevel);
+                    }
+                    if (($tblLevel = Division::useService()->getLevelBy($Data->getTblSchoolTypeSecondary(), '9'))) {
+                        $Data->createCertificateLevel($tblCertificate, $tblLevel);
+                    }
+                }
+            }
+            // Begrenzung der Einschätzung
+            $FieldName = 'Rating';
+            if (!$Data->getCertificateFieldByCertificateAndField($tblCertificate, $FieldName)) {
+                $Data->createCertificateField($tblCertificate, $FieldName, 2500);
+            }
+            // Begrenzung Bemerkungsfeld
+            $FieldName = 'RemarkWithoutTeam';
+            if (!$Data->getCertificateFieldByCertificateAndField($tblCertificate, $FieldName)) {
+                $Data->createCertificateField($tblCertificate, $FieldName, 1000);
+            }
+            if (!$Data->getCertificateSubjectAll($tblCertificate)) {
+                $Data->setCertificateSubject($tblCertificate, 'DE', 1, 1);
+                $Data->setCertificateSubject($tblCertificate, 'EN', 1, 2);
+                $Data->setCertificateSubject($tblCertificate, 'KU', 1, 3);
+                $Data->setCertificateSubject($tblCertificate, 'MU', 1, 4);
+                $Data->setCertificateSubject($tblCertificate, 'GE', 1, 5);
+                $Data->setCertificateSubject($tblCertificate, 'GRW', 1, 6);
+                $Data->setCertificateSubject($tblCertificate, 'GEO', 1, 7);
+                $Data->setCertificateSubject($tblCertificate, 'WTH', 1, 8);
+
+                $Data->setCertificateSubject($tblCertificate, 'MA', 2, 1);
+                $Data->setCertificateSubject($tblCertificate, 'BIO', 2, 2);
+                $Data->setCertificateSubject($tblCertificate, 'CH', 2, 3);
+                $Data->setCertificateSubject($tblCertificate, 'PH', 2, 4);
+                $Data->setCertificateSubject($tblCertificate, 'INF', 2, 5);
+                $Data->setCertificateSubject($tblCertificate, 'TUC', 2, 6);
+                $Data->setCertificateSubject($tblCertificate, 'RE/e', 2, 7);
+                $Data->setCertificateSubject($tblCertificate, 'SPO', 2, 8);
+            }
+        }
+    }
+    /**
+     * @param Data        $Data
+     * @param TblConsumer $tblConsumerCertificate
+     */
+    private static function setEzshGymHj(Data $Data, TblConsumer $tblConsumerCertificate)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setEzshGymHj() has a Cyclomatic Complexity of 12. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)